src/app/public/modules/popover/popover.component.html

```diff
@@ -1,52 +1,5 @@
-<div
-  class="sky-popover-container"
-  tabindex="-1"
```

Comment:
The current popover is not focusable. It looks like this change may have made them focusable - was that deliberate?

Reply:
See above.